proof-of-concept/test_pof.sh

```diff
@@ -105,6 +105,24 @@ setup_py_install() {
 
 # ======== main code =======
 
+# validate arguments
+if [[ "$#" -lt 1  || ( "$#" -lt 2  && "$1" != "clean") ]]; then
+  echo "Usage: $0 COMMAND [TARGET_ABI]" >&2
+  echo "Commands:" >&2
+  echo "  wheel TARGET_ABI: build a wheel, install and test"
+  echo "  setup_py_install TARGET_ABI: install poc using 'setup.py install' & run tests" >&2
+  echo "  clean: clean build artifacts" >&2
```

Comment:
I added another command in #143, namely `setup_py_build_ext_inplace`, but I can add the documentation for that after this PR is merged.